hotfixes/hotfix-sp-repair

```diff
@@ -141,15 +141,74 @@ repairBSLServices() {
 	fi
 }
 
+repairNomadParentContainer() {
+	echo running repairNomadParentContainer
+	npc_image_local="nomad-parent-container:local"
+	npc_image_backup="nomad-parent-container:backup"
+
+	if [[ $(docker images --format "{{.ID}}" $npc_image_local | wc -l) -gt 0 ]]; then
+		echo "$npc_image_local already installed. Skipping to repair $npc_image_local..."
+		return
+	fi
+
+	docker_registry="$(jq -r '.managementHttpsEndpoint' /opt/bluecat/bsld/conf/node.conf):443/library"
+	npc_image_461="$docker_registry/nomad-parent-container:1.4.12-build.381104-d8a94958"
+
+	docker pull "$npc_image_461"
+	if [[ $(docker images --format "{{.ID}}" $npc_image_461 | wc -l) -eq 0 ]]; then
+		echo "Unable to download docker image $npc_image_461. Exiting..."
+		exit 1
+	fi
+
+	docker tag "$npc_image_461" "$npc_image_local"
+	docker tag "$npc_image_461" "$npc_image_backup"
+	docker rmi $npc_image_461
+}
+
+installBSLD() {
+	echo installing BSLD
+	package="sdp-bsld=$1"
+
+	apt update && apt install --reinstall -y $package
+
+	# verify install
+	installed=$(apt list --installed sdp-bsld 2>/dev/null -qq | sed 's/\(.*\)\/[^ ]* \([^ ]*\) .*/\1=\2/')
+	if [[ $installed != $package ]]; then
+		echo "Expected BSLD package is not installed on the system"
+		echo "BSLD Version: $installed"
```

Comment:
if the version is wrong, would it help to print a checksum as we've done elsewhere in this hotfix?

Reply:
The checksum would only help if we have the same target and installed version. We could modify it so that it prints the checksum regardless of the installed version.